docs/apache-airflow/faq.rst

```diff
@@ -522,3 +522,27 @@ This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the ``mysqld`` section in your ``my.cnf`` file.
 #. Restart the Mysql server.
+
+Does Airflow collect any telemetry data?
```

Comment:
i think this is too easily confused with otel / stats

Reply:
maybe better to articulate as "usage tracking" or "analytics" or "usage analytics" or... something

Reply:
It can be yes but the description should help clear things up